panic: runtime error: invalid memory address or nil pointer dereference

Code from demo:
```Go
package main

import (
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	app := app.New()

	w := app.NewWindow("Hello")
	w.SetContent(widget.NewVBox(
		widget.NewLabel("Hello Fyne!"),
		widget.NewButton("Quit", func() {
			app.Quit()
		}),
	))

	w.ShowAndRun()
}
```
```
go run demo.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x51bb82]

goroutine 1 [running, locked to thread]:
fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw.(*Window).MakeContextCurrent.func1(0x0)
        /home/user/Go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/context.go:16 +0x22
fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw.(*Window).MakeContextCurrent(0x0)
        /home/user/Go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/context.go:16 +0x2b
fyne.io/fyne/driver/gl.(*gLDriver).CreateWindow.func1()
        /home/user/Go/src/fyne.io/fyne/driver/gl/window.go:779 +0x11c
fyne.io/fyne/driver/gl.runOnMain(0xc0000866c0)
        /home/user/Go/src/fyne.io/fyne/driver/gl/loop.go:39 +0xab
fyne.io/fyne/driver/gl.(*gLDriver).CreateWindow(0xc00009a700, 0x6599a6, 0x5, 0xc0000920c0, 0x7f8900)
        /home/user/Go/src/fyne.io/fyne/driver/gl/window.go:763 +0x99
fyne.io/fyne/app.(*fyneApp).NewWindow(0xc0000920c0, 0x6599a6, 0x5, 0xc0000920c0, 0xc0000a9f88)
        /home/user/Go/src/fyne.io/fyne/app/app.go:33 +0x47
main.main()
        /tmp/demo.go:11 +0xb9
exit status 2
```

@andydotxyz 
```
2019/03/13 21:16:13 Window creation error VersionUnavailable: GLX: Failed to create context: GLXBadFBConfig
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x59c226]

goroutine 1 [running, locked to thread]:
fyne.io/fyne/driver/gl.(*window).SetContent(0x0, 0x6a2f00, 0xc00005e420)
        /home/user/Go/src/fyne.io/fyne/driver/gl/window.go:312 +0x26
main.main()
        /tmp/app.go:12 +0x21e
exit status 2
```

--- a/driver/gl/gl.go
+++ b/driver/gl/gl.go
@@ -304,10 +304,10 @@ func compileShader(source string, shaderType uint32) (uint32, error) {
 
 const (
 	vertexShaderSource = `
-    #version 150
-    in vec3 vert;
-    in vec2 vertTexCoord;
-    out vec2 fragTexCoord;
+    #version 110
+    attribute vec3 vert;
+    attribute vec2 vertTexCoord;
+    varying vec2 fragTexCoord;
 
     void main() {
         fragTexCoord = vertTexCoord;
@@ -317,14 +317,13 @@ const (
 ` + "\x00"
 
 	fragmentShaderSource = `
-    #version 150
+    #version 110
     uniform sampler2D tex;
 
-    in vec2 fragTexCoord;
-    out vec4 frag_colour;
+    varying vec2 fragTexCoord;
 
     void main() {
-        frag_colour = texture(tex, fragTexCoord);
+        gl_FragColor = texture2D(tex, fragTexCoord);
     }
 ` + "\x00"
 )

--- a/driver/gl/window.go
+++ b/driver/gl/window.go
@@ -770,10 +770,8 @@ func (d *gLDriver) CreateWindow(title string) fyne.Window {
 		// make the window hidden, we will set it up and then show it later
 		glfw.WindowHint(glfw.Visible, 0)
 
-		glfw.WindowHint(glfw.ContextVersionMajor, 3)
-		glfw.WindowHint(glfw.ContextVersionMinor, 2)
-		glfw.WindowHint(glfw.OpenGLProfile, glfw.OpenGLCoreProfile)
-		glfw.WindowHint(glfw.OpenGLForwardCompatible, glfw.True)
+		glfw.WindowHint(glfw.ContextVersionMajor, 2)
+		glfw.WindowHint(glfw.ContextVersionMinor, 0)
 
 		win, err := glfw.CreateWindow(10, 10, title, nil, nil)
 		if err != nil {

